packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1355,6 +1356,21 @@ async function renderToHTMLOrFlightImpl(
   const buildFailingError =
     digestErrorsMap.size > 0 ? digestErrorsMap.values().next().value : null
 
+  // If we're debugging partial prerendering, print all the dynamic API accesses
+  // that occurred during the render.
+  if (
+    staticGenerationStore.prerenderState &&
+    usedDynamicAPIs(staticGenerationStore.prerenderState) &&
+    staticGenerationStore.prerenderState?.isDebugSkeleton
```

Comment:
I'm confused about how this is supposed to work. We only have a prerenderState during build (and revalidate) but we shouldn't have a `__nextppronly` query param at this time so won't this be false anytime the prerenderState exists?

Reply:
The `__nextppronly` query parameter is a stop-gap until we have more comprehensive debugging available for suspense and/or partial prerendering. While running with `next dev`, when partial prerendering is enabled, you can show the static shell only. This pull request extends this to also print the Dynamic API's used that influenced the static shell generation. Any Dynamic API not called during the static shell generation doesn't affect it (yet).

Reply:
So in Dev the prerenderState exists when the query param is set?